appinventor/codelabs/mole-mash/index.html

```diff
@@ -0,0 +1,242 @@
+          </ul>
+        <p>Your app should look something like this:</p>
+        <img src="img/ui1.png">
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Add a timer">
+        <p>In order to make the mole jump periodically, you'll need a timer. Add a <b>Clock</b> component and name it "MoleTimer"; this component will go into the <b>non-visible components</b> area. Set the <b>TimeInterval</b> property to 500 milliseconds, and make sure that <b>TimerEnabled</b> is checked.</p>
+        <img src="img/timer.png">
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Add a mole">
+        <p>To add the moving mole, we'll use a <b>sprite</b>. Sprites are images that can move on the screen within a Canvas. For MoleMash, our sprite won't move by itself; instead, you'll be setting the mole's position each time the timer fires.</p>
+        <p>From the <b>Drawing and Animation</b> drawer, drag a <b>ImageSprite</b> into your workspace, within the area of <b>MyCanvas</b>. Name this component "Mole", and set these properties for your sprite:</p>
+          <ul>
+            <li>Picture: Use mole.png, which you downloaded to your computer at the beginning of this tutorial.</li>
+            <li>Enabled: checked</li>
+            <li>Interval: 500 (The interval doesn't matter here, because the mole's speed is zero: it's not moving by itself.)</li>
+            <li>Heading: 0 The heading doesn't matter here either, because the speed is 0.</li>
+            <li>Speed: 0.0</li>
+            <li>Visible: checked</li>
+            <li>Width: Automatic</li>
+            <li>Height: Automatic</li>
+          </ul>
+        <aside class="callout">
+          <h2>Test your app</h2>
+          <p>You should see that the <b>x</b> and <b>y</b> properties of the Mole sprite are already filled in. Drag the Mole around on your Canvas some more. How have the <b>x</b> and <b>y</b> properties changed? How are these properties determined?</p>
+        </aside>
+        <p>Your user interface should look something like this:</p>
+        <img src="img/ui2.png">
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Checkoff  #1">
+        <aside class="special">
+          <h2>mole-mash-1</h2>
+          <p><b>To receive this checkoff, you must</b></p>
+          <ul class="checklist">
+            <li>Show your finished User Interface</li>
+            <li>Explain what the TimerInterval property of the Clock component does</li>
+            <li>Describe what a sprite component is, and how it's used in your app</li>
+          </ul>
+        </aside>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Define a procedure to move the mole">
+        <p>Now it's time to code your app. To do that, we're going to need a new type of block: <i>procedures</i>. A procedure is a sequence of statements that you can refer to all at once as single command.</p>
```

Comment:
Add an `aside class="note"` for the procedure definition. Would be helpful to explain why procedures are useful and why we use them.